code-scanning/frogbot-fix.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# Read more about Frogbot here - https://github.com/jfrog/frogbot#frogbot
+
+name: "Frogbot Fix"
+on:
+  push:
+    branches:
+      # The scanning and creation of pull requests with fixes are triggered by pushing code to one of the these branches.
+      # You can edit the list of branches you wish to open fix pull requests on.
+      - "main"
+      - "master"
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  create-fix-pull-requests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      # Install prerequisites - uncomment the relevant one
+
+      # - uses: actions/setup-go@v3
+      #   with:
+      #     go-version: 1.17.x
+
+      # - uses: actions/setup-java@v3
+      #    with:
+      #      java-version: "11"
+      #      distribution: "temurin"
+
+      # - uses: actions/setup-node@v3
+      #    with:
+      #      node-version: "16.x"
+
+
+      - uses: jfrog/frogbot@v2
```

Comment:
Can we use full 40 character commit sha instead? We prefer that for security reasons.

Reply:
Does it create a sarif file? If yes does it upload in the action itself?

Reply:
Yes, it creates and upload the file by itself:
https://github.com/jfrog/froggit-go/blob/master/vcsclient/github.go#L446

Reply:
Does it not need `security-events: write # for github/codeql-action/upload-sarif to upload SARIF results`  in permissions?

Reply:
Hi @Phantsure, the upload mechanism is handled by Frogbot using the code I sent you, so it does not require those permissions. The upload-sarif action is not being used.

Reply:
commit-sha replaced the tag now :)

Reply:
@omerzi Can you point me in the direction where the upload is happening? Our code-scanning workflows generally have uploads to any of the security interfaces

Reply:
@Phantsure, of course:
https://github.com/jfrog/frogbot/blob/master/commands/utils/utils.go#L74
Which uses:
https://github.com/jfrog/froggit-go/blob/master/vcsclient/github.go#L446